### ISSUE TITLE
Event decks do not properly load from save.

### ISSUE THREAD

**Issue by jhcheung** [Sebaestschjin/gloomhaven-campaign-manager#66]
![road events does not exist](https://user-images.githubusercontent.com/4327784/95157263-80915980-0766-11eb-8f9e-2333fb906158.png)

Using this event code:
`    "events": [
    {
      "deck": "Road Events",
      "add": [
        31
      ],
      "remove": [
        22,
        5,
        18,
        17,
        12,
        26
      ],
      "bottomUp": [
        1,
        27,
        9,
        14,
        3,
        2,
        29,
        13,
        10,
        31,
        21,
        28,
        15,
        16,
        17,
        24,
        30,
        7,
        20,
        6,
        23,
        11,
        25,
        8,
        4
      ]
    },
    {
      "deck": "City Events",
      "add": [
        31,
        78
      ],
      "remove": [
        16,
        12,
        22,
        7,
        3,
        4,
        5,
        6,
        31,
        23
      ],
      "bottomUp": [
        17,
        14,
        1,
        29,
        10,
        11,
        24,
        28,
        18,
        30,
        26,
        9,
        20,
        25,
        15,
        78,
        19,
        8,
        2,
        21,
        13,
        27
      ]
    }
  ]
`

Note that it also seems like saving a new file no longer includes an event deck property & section.

**Comment by Sebaestschjin** [comment]
That's weird. The created save has the wrong names for road and city events. It should be `"deck": "road"` and `"deck": "city"` instead of the full name. Was this created by the mod?
I couldn't reproduce this, yet and also couldn't find anything in the code that would produce those names. Unless there's a very weird thing happening with Lua here, because there are two variables named `name`. One stands for `road` and the other for `Road Events`, but they aren't and shouldn't be related.

> Note that it also seems like saving a new file no longer includes an event deck property & section.
Do you mean the only the `deck` property is missing or all properties (like `add` and `remove`)?

**Comment by jhcheung** [comment]
> 
> 
> That's weird. The created save has the wrong names for road and city events. It should be `"deck": "road"` and `"deck": "city"` instead of the full name. Was this created by the mod?
> I couldn't reproduce this, yet and also couldn't find anything in the code that would produce those names. Unless there's a very weird thing happening with Lua here, because there are two variables named `name`. One stands for `road` and the other for `Road Events`, but they aren't and shouldn't be related.

Ah, this makes sense. The save was not generated by the mod itself. I manually generated this save using the recommended schema and JSON editor in the ['How to use' instructions](https://github.com/Sebaestschjin/gloomhaven-campaign-manager/wiki/How-to-use) because I was not able to save my events via the mod itself. Seems like there was a typo in the schema.

> 
> > Note that it also seems like saving a new file no longer includes an event deck property & section.
> 
> Do you mean the only the `deck` property is missing or all properties (like `add` and `remove`)?

I figured this issue out. It turns out that I was leaving my Road/City events in the default piles (the Road Events 1-30 and City Events 1-30), and did not move it to the piles near the center of the table. Might be good to document this in the 'How to use' or maybe provide an error if the events are not in the appropriate place.

**Comment by Sebaestschjin** [comment]
Yeah, the values in the schema were wrong for events. Fixed that now and also added the info that the event decks have to be on their mats. Thanks for your help. :-)
